extensions/proposals/WEBGL_video_texture/extension.xml

```diff
@@ -52,13 +52,11 @@ interface WebGLVideoFrameInfo {
 };
 
 [NoInterfaceObject]
-interface OES_EGL_image_external {
-    const GLenum TEXTURE_EXTERNAL_OES             = 0x8D65;
-    const GLenum SAMPLER_EXTERNAL_OES             = 0x8D66;
-    const GLenum TEXTURE_BINDING_EXTERNAL_OES     = 0x8D67;
-    const GLenum REQUIRED_TEXTURE_IMAGE_UNITS_OES = 0x8D68;
+interface WEBGL_video_texture {
+    const GLenum TEXTURE_VIDEO_IMAGE             = 0x851D;
+    const GLenum SAMPLER_VIDEO_IMAGE             = 0x8B61;
```

Comment:
May be need also add the following enum values?

TEXTURE_BINDING_VIDEO_IMAGE_WEBGL = 0x924A
glGet() params returns a single value, the name of the texture currently bound to the target GL_TEXTURE_VIDEO_IMAGE_WEBGL

REQUIRED_TEXTURE_VIDEO_IMAGE_UNITS_WEBGL = 0x924B
For platforms which will use underlying TEXTURE_EXTERNAL_OES for new texture target we must know
how much texture image units needed for video texture.

According to https://www.khronos.org/registry/OpenGL/extensions/OES/OES_EGL_image_external.txt
Each TEXTURE_EXTERNAL_OES texture object may require up to 3 texture image units for each texture unit to which it is bound.